### PR TITLE
Updated email sign in template

### PR DESCRIPTION
**Pull request by n4t3m**
the email that is sent to to you when you try to sign-in via email is not terrible now

example email: https://cdn.discordapp.com/attachments/997701178240536639/1017003056468086784/unknown.png

### CODE REVIEW

File: pages/api/auth/[...nextauth].ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import nodemailer from 'nodemailer'
```
Comment (minsooerickim) [inline pages/api/auth/[...nextauth].ts]: can u do import createTransport from 'nodemailer'?

File: pages/api/auth/[...nextauth].ts [hunk 2]
```diff
@@ -68,3 +84,94 @@ export default async function auth(req: NextApiRequest, res: NextApiResponse) {
     secret: process.env.SECRET,
   })
 }
+
+function html({ url, host, email }) {
+  const escapedEmail = `${email.replace(/\./g, '&#8203;.')}`
+  const escapedHost = `${host.replace(/\./g, '&#8203;.')}`
+  return `
+  <html>
```
Comment (minsooerickim) [inline pages/api/auth/[...nextauth].ts]: can u move this out to a constants file or sum. i would like to keep nextauth file as clean as poss

File: pages/api/auth/[...nextauth].ts (hunk 1 above)
Comment (n4t3m) [reply pages/api/auth/[...nextauth].ts]: done

File: pages/api/auth/[...nextauth].ts (hunk 2 above)
Comment (n4t3m) [reply pages/api/auth/[...nextauth].ts]: done, moved these functions to a different file to be more clean

File: lib/verificationMailContent.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+                      <p>
+                         Head back to our website at <a href="${host}">${host}</a>
+                      </p>
+                      <p>
+                         Contact us, whenever at <a href="mailto:citrushack@gmail.com">citrushack@gmail.com</a>
+                      </p>
+                      <a href="http://localhost:3000"><img class="social" src="http://cdn.mcauto-images-production.sendgrid.net/cbb380edd52c8d3a/18518ec5-9e99-4737-8748-aaf321c86d93/24x24.png"/></a>
+                      <a href="http://localhost:3000"><img class="social" src="http://cdn.mcauto-images-production.sendgrid.net/cbb380edd52c8d3a/6b62002f-f863-4ca1-9f6c-c11ee2cef453/24x24.png" /></a>
+                      <a href="http://locahost:3000"><img class="social" src="http://cdn.mcauto-images-production.sendgrid.net/cbb380edd52c8d3a/5bc8e9f4-455c-4f40-b6aa-cc552e812c04/24x24.png" /></a>
+                      <a href="http://localhost:3000"><img class="social" src="http://cdn.mcauto-images-production.sendgrid.net/cbb380edd52c8d3a/0061c602-b7f9-4a2c-a015-66c3aa221f33/24x24.png" /></a>
```
Comment (minsooerickim) [inline lib/verificationMailContent.js]: shud probably use `host` instead of hardcoding in localhost?

Comment (n4t3m) [reply lib/verificationMailContent.js]: these are all placeholder values!!!

File: pages/api/auth/[...nextauth].ts (hunk 2 above)
Comment (minsooerickim) [reply pages/api/auth/[...nextauth].ts]: NAB, but u think u can make it a ts file? 😂😂😂 sry

File: lib/verificationMailContent.js (hunk 3 above)
Comment (minsooerickim) [reply lib/verificationMailContent.js]: right but it shud still use host. the value we pass in shud be whats localhost for the placeholder val

**Pull request merged**